### ISSUE TITLE
.decode('utf-8') in connection.py is not needed for Python 3.x

### ISSUE THREAD

**Issue by Eugnis** [timotheus/ebaysdk-python#136]
Receiving error on Python 3.4 while using Visual Studio only. Begin with Python 3, all string is unicode object.
   ```
 response = self.api.execute('AddMemberMessageAAQToPartner', message_obj)
  File "\ebaysdk\connection.py", line 123, in execute
    self.error_check()
  File "\ebaysdk\connection.py", line 209, in error_check
    estr = self.error()
  File "\ebaysdk\connection.py", line 325, in error
    error_array = [smart_encode(e).decode('utf-8') for e in error_array]
  File "\ebaysdk\connection.py", line 325, in <listcomp>
    error_array = [smart_encode(e).decode('utf-8') for e in error_array]
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by gumbert** [comment]
I have the same problem. Windows 10 x64, my python

```
c:\docs\gdrive\docs\prog\ebay>python
Python 3.5.1 |Anaconda 2.4.1 (64-bit)| (default, Dec  7 2015, 15:00:12) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
```
error

```
c:\docs\gdrive\docs\prog\ebay>python try.py
Traceback (most recent call last):
  File "try.py", line 7, in <module>
    api.execute('findCompletedItems', {'categoryId': '619'})
  File "C:\Anaconda3\lib\site-packages\ebaysdk\connection.py", line 123, in execute
    self.error_check()
  File "C:\Anaconda3\lib\site-packages\ebaysdk\connection.py", line 209, in error_check
    estr = self.error()
  File "C:\Anaconda3\lib\site-packages\ebaysdk\connection.py", line 325, in error
    error_array = [smart_encode(e).decode('utf-8') for e in error_array]
  File "C:\Anaconda3\lib\site-packages\ebaysdk\connection.py", line 325, in <listcomp>
    error_array = [smart_encode(e).decode('utf-8') for e in error_array]
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by Eugnis** [comment]
Solved that by removing `.decode('utf-8')` here
` File "\ebaysdk\connection.py", line 325, in error
    error_array = [smart_encode(e).decode('utf-8') for e in error_array]`

**Comment by bourdibay** [comment]
Actually this has been fixed in the commit #[995947c6a23e8907a4928adc716445bc9873aaa8](https://github.com/timotheus/ebaysdk-python/commit/995947c6a23e8907a4928adc716445bc9873aaa8)
You should checkout the most up-to-date version of the repo.

**Comment by gumbert** [comment]
> Actually this has been fixed in the commit #995947c6a23e8907a4928adc716445bc9873aaa8
> You should checkout the most up-to-date version of the repo.

Thanks bourdibay, that help me.
